certified-connectors/CDK Drive Service Vehicles/apiProperties.json

```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#007ee5",
```

Comment:
This is the default icon brand color, which unfortunately isn't allowed (or #ffffff), so this will need to be changed